### PR TITLE
CRAN, pharmaverse badges

### LINKED ISSUE #5: General Issue: add badges
### Background Information

Add pharmaverse badge and any other badges (covr, etc.)

- https://pharmaverse.org/contribute/badges/
- [](https://pharmaverse.org)

### Definition of Done

Badges are shown on readme and pkgdown

### PR DESCRIPTION
**Pull request by nicholas-masel**
Closes #5  

Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/articles/development_process.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the codebase remains robust and consistent.

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests - See [Unit Test Guide](https://pharmaverse.github.io/admiral/articles/unit_test_guidance.html#writing-unit-tests-in-admiral-)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiral/articles/programming_strategy.html#deprecation-1)?
- [x] Update to all relevant roxygen headers and examples.
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiral/reference/index.html)" page.
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue so that it closes after successful merging.
- [x] Address all merge conflicts and resolve appropriately.
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

**Comment by nicholas-masel** [comment]
@mstackhouse What do you know about codecov reposistories?   Do we have to set this up on app.codecov.io/gh/pharmaverse prior to leveraging the github action?

[Here is the error](https://github.com/pharmaverse/envsetup/actions/runs/3159771648/jobs/5143414522#step:5:43 
)

**Comment by mstackhouse** [comment]
@cicdguy would you be able to give us some guidance to getting code coverage set up for this repository?

File: .github/workflows/test-coverage.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+          use-public-rspm: true
+
+      - uses: r-lib/actions/setup-r-dependencies@v2
+        with:
+          extra-packages: any::covr
+          needs: coverage
+
+      - name: Test coverage
+        run: covr::codecov(quiet = FALSE, function_exclusions = "\\.onLoad")
+        shell: Rscript {0}
```
[the remark this answers is not in the document]
Comment (nicholas-masel) [reply .github/workflows/test-coverage.yaml]: Thanks @mstackhouse and @cicdguy.  I swapped the `common.yaml` back in, and I get an error that `DT` and `htmltools` packages must be installed to use `covr::report()`.  Can we add this to the workflow or do these have to be package dependencies?

https://github.com/pharmaverse/envsetup/actions/runs/3759304585/jobs/6388708844#step:12:70

[the remark this answers is not in the document]
Comment (nicholas-masel) [reply .github/workflows/test-coverage.yaml]: Thanks @cicdguy!  It is no longer failing.

**Pull request merged**